### PR TITLE
refactor: read service list from GitHub

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: R/emodnet_wfs.R
Comment (maelle) [inline R/emodnet_wfs.R:14]: this and the other `getOption()` are escape hatches https://blog.r-hub.io/2023/01/23/code-switch-escape-hatch-test/

we could also maybe document them, but setting them because GitHub is down is really "advanced".

File: R/emodnet_wfs.R
Comment (maelle) [inline R/emodnet_wfs.R:31]: although if GitHub is down, opening an issue might not work :upside_down_face:

File: R/emodnet_wfs.R
Comment (maelle) [inline R/emodnet_wfs.R:53]: #154

File: R/emodnet_wfs.R
Comment (salvafern) [reply R/emodnet_wfs.R:14]: They seem fine for me :)

**Review by salvafern — APPROVED** [review]

File: R/emodnet_wfs.R
Comment (maelle) [inline R/emodnet_wfs.R:15]: @salvafern should we wait until your PR is merged until we merge?

File: R/emodnet_wfs.R
Comment (maelle) [inline R/emodnet_wfs.R:45]: or should we keep this name? how should service names be deduced? @salvafern

File: R/emodnet_wfs.R
Comment (salvafern) [reply R/emodnet_wfs.R:15]: Probably yes..

**Pull request closed without merging**
